### PR TITLE
Update dlp-policy-reference.md

### PR DESCRIPTION
**Pull request by edwinchan-microsoft**
added Preview items for Trainable classifiers

@chrfox @LauraWi

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: microsoft-365/compliance/dlp-policy-reference.md [hunk 1]
```diff
@@ -130,6 +130,9 @@ DLP policies detect sensitive items by matching them to a sensitive information
 |On-Premises repositories| Yes| Yes| No|
 |PowerBI|Yes | Yes| No|
 
+> [!PREVIEW]
+> DLP now supports usage of Trainable Classifiers as a condition to detect sensitive documents. Content can be defined by Trainable Classifiers in Exchange email online, Sharepoint Online sites, OneDrive for Business accounts, Teams Chat and Channels, and Devices. See, [Trainable Classifiers](classifier-learn-about.md)
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-policy-reference.md]:
```suggestion
> DLP now supports the usage of Trainable Classifiers as a condition to detect sensitive documents. Content can be defined by Trainable Classifiers in Exchange email online, Sharepoint Online sites, OneDrive for Business accounts, Teams Chat and Channels, and Devices. For more information, see [Trainable Classifiers](classifier-learn-about.md).
```

File: microsoft-365/compliance/dlp-policy-reference.md [hunk 2]
```diff
@@ -211,6 +214,7 @@ Conditions are inclusive and are where you define what you want the rule to look
 - [sensitive information types](sensitive-information-type-learn-about.md#learn-about-sensitive-information-types)
 - [sensitivity labels](sensitivity-labels.md)
 - [retention labels](retention.md#using-a-retention-label-as-a-condition-in-a-dlp-policy)
+- (In Preview)[Trainable Classifiers](classifier-learn-about.md)
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-policy-reference.md]:
```suggestion
- [Trainable Classifiers](classifier-learn-about.md) (in preview) 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @edwinchan-microsoft, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by KCCross** [comment]
KC merging on behalf of Chris Fox. Update DLP policy reference for using TC as a condition is now in public preview.

**Pull request merged**